### PR TITLE
Update translations for Traditional Chinese

### PR DESCRIPTION
**Pull request by rhybs**
Update translations for Traditional Chinese.

### CODE REVIEW

**Review by Xazin — APPROVED** [review]

File: frontend/resources/translations/zh-TW.json [hunk 1]
```diff
@@ -614,7 +616,7 @@
       "textPlaceholder": "空白",
       "copyProperty": "已將屬性複製到剪貼簿",
       "count": "計數",
-      "newRow": "新行",
+      "newRow": "新增列",
```
Comment (appflowy) [inline frontend/resources/translations/zh-TW.json:619]: should be "新增行"

Comment (rhybs) [reply frontend/resources/translations/zh-TW.json]: The meaning of rows and columns is opposite between zh-TW and zh-CN.
[ref](https://terms.naer.edu.tw/detail/5e94c72b55cc73ebf81d71ca2b5d57e6/?seq=1)

Comment (rhybs) [reply frontend/resources/translations/zh-TW.json]: @appflowy so it should be "新增列"

Comment (richardshiue) [reply frontend/resources/translations/zh-TW.json]: Hi @rhybs, can you open a new PR with these changes?

Comment (rhybs) [reply frontend/resources/translations/zh-TW.json]: > Hi @rhybs, can you open a new PR with these changes?

Hi @richardshiue, these changes are in #4683 .

Comment (richardshiue) [reply frontend/resources/translations/zh-TW.json]: @rhybs Got it, thanks for contributing!

**Pull request merged**